File: order-service/src/main/java/com/intexsoft/webshop/orderservice/service/OrderService.java
package com.intexsoft.webshop.orderservice.service;

import com.intexsoft.webshop.orderservice.dto.order.OrderCreateDto;
import com.intexsoft.webshop.orderservice.dto.order.OrderDto;

public interface OrderService {
    OrderDto saveOrder(OrderCreateDto orderCreateDto);
}

File: order-service-kt/src/main/kotlin/com/intexsoft/webshop/orderservicekt/service/OrderService.kt
package com.intexsoft.webshop.orderservicekt.service

import com.intexsoft.webshop.orderservicekt.dto.order.OrderCreateDto
import com.intexsoft.webshop.orderservicekt.dto.order.OrderDto

interface OrderService {
    fun saveOrder(orderCreateDto: OrderCreateDto): OrderDto
}